### PR TITLE
doc: fixed restful mgr module SSL configuration commands

### PR DESCRIPTION
**Pull request by LenzGr**
Fixed restful mgr module SSL configuration commands

Fixed typo in a reStructuredText note entity of the iSCSI config chapter

Signed-off-by: Lenz Grimmer <lgrimmer@suse.com>

### CODE REVIEW

**Comment by LenzGr** [comment]
jenkins render docs

File: doc/mgr/restful.rst [hunk 1]
```diff
@@ -40,15 +40,15 @@ can be generated with a command similar to::
 The ``restful.crt`` should then be signed by your organization's CA
 (certificate authority).  Once that is done, you can set it with::
 
-  ceph config set mgr mgr/restful/$name/crt -i restful.crt
-  ceph config set mgr mgr/restful/$name/key -i restful.key
+  ceph config-key set mgr mgr/restful/$name/crt -i restful.crt
```
Comment (jcsp) [inline doc/mgr/restful.rst]: Should be "ceph config key set mgr/restful..." (no extra "mgr" bit between).

The extra "mgr" bit was new with "ceph config set", specifying which daemon the config setting applies to, whereas with "config-key" it's all just one global store.

Comment (LenzGr) [reply doc/mgr/restful.rst]: Fixed, thanks for the clarification!

**Comment by LenzGr** [comment]
jenkins render docs

**Comment by liewegas** [comment]
This targets master; changed milestone.

**Pull request merged**